tests/DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.UnitTests/Repositories/AcademyRepositoryOfstedTests.cs

```diff
@@ -848,7 +841,10 @@ public async Task
             $"URN {giasGroupLinks[8].Urn!} has a previous Ofsted single headline grade of Inadequate issued"
             // giasGroupLinks[9] - OverallEffectiveness before policy change date so no error log expected
         );
-        _mockLogger.VerifyNoOtherCalls();
+
+        var urnId = int.Parse(giasGroupLinks[9].Urn!);
+
+        VerifyLogs([urnId], shouldLogError: false);
```

Comment:
This could give a false positive as `VerifyLogs` checks for the particular message `"URN {urn} has some unrecognised ofsted ratings. This could be a data integrity issue with the Ofsted data in Academies Db."`
We could instead use `VerifyDidNotReceive<T>(this ILogger<T> logger, string regexPattern)` with just the URN